src/volue/mesh/_base_session.py

```diff
@@ -805,6 +805,46 @@ def update_rating_curve_versions(
             :doc:`mesh_rating_curve`
         """
 
+    @abc.abstractmethod
+    def run_simulation(
+        self,
+        model: str,
+        case: str,
+        start_time: datetime,
+        end_time: datetime,
+        resolution: Timeseries.Resolution,
+        scenario: int,
+        return_datasets: bool,
+    ) -> Union[typing.Iterator[None], typing.AsyncIterator[None]]:
+        """Run a hydro simulation using HydSim on the Mesh server.
+
+        Args:
+            model: The name of the Mesh model in which the simulation case exists.
+            case: The names of the case group and simulation case on the form
+                'CaseGroup/CaseName'.
```

Comment:
```suggestion
            case: The names of the case group and simulation case in the form
                'CaseGroup/CaseName'.
```

I think should be "in the form". Or change it to something like "in the following format".

Reply:
Fixed, thanks!